trpc/router/goalHistory.ts

```diff
@@ -0,0 +1,28 @@
+import { TRPCError } from '@trpc/server';
```

Comment:
I'm not sure we really need this endpoint. We will use prisma methods in other routes.